### PR TITLE
Added Spotify.Spotify version 1.1.63.568.gda8cb5ac

### PR DESCRIPTION
**Pull request by ImJoakim**
image

```
PS C:\Users\WDAGUtilityAccount\Desktop\Git> winget install -m "C:\Users\WDAGUtilityAccount\Desktop\Git\winget-pkgs\manifests\s\Spotify\Spotify\1.1.63.568.gda8cb5ac"
Found Spotify [Spotify.Spotify]
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://download.scdn.co/SpotifySetup.exe
  ██████████████████████████████   878 KB /  878 KB
Successfully verified installer hash
Starting package install...
Successfully installed
```

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/20701)

### CODE REVIEW

File: manifests/s/Spotify/Spotify/1.1.63.568.gda8cb5ac/Spotify.Spotify.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+- InstallerLocale: en-US
+  Architecture: neutral
+  InstallerType: exe
+  Scope: user
+  InstallerUrl: https://download.scdn.co/SpotifySetup.exe
+  InstallerSha256: BFEAF82427E966ECFEC792CEE8B95C8A4AC077BEDB923BC337DE7C6DA9314186
+  InstallerSwitches:
+    Silent: ' '
+    SilentWithProgress: ' '
+  UpgradeBehavior: install
```
Comment (OfficialEsco) [inline manifests/s/Spotify/Spotify/1.1.63.568.gda8cb5ac/Spotify.Spotify.installer.yaml:21]:
```suggestion
MinimumOSVersion: 10.0.0.0
InstallModes:
- silent
- silentWithProgress
Installers:
- Platform:
  - Windows.Desktop
  InstallerLocale: en-US
  Architecture: neutral
  InstallerType: exe
  Scope: user
  InstallerUrl: https://download.scdn.co/SpotifyFullSetup.exe
  InstallerSha256: E86EA6110AB5FE30B99807FFE150E41031EC52C683A3F7A51D638601D1F4F9E3
  InstallerSwitches:
    Silent: /Silent
    SilentWithProgress: ' '
  UpgradeBehavior: install
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Hmm should we use the Offline installer instead? The /Silent switch somewhat works on it, wish we could stop the auto launch tho..

Also just found out
https://download.scdn.co/SpotifySetup.exe (1.1.63.568)
https://download.scdn.co/SpotifyFullSetup.exe (1.1.63.568)
and 
http://download.spotify.com/SpotifySetup.exe (1.1.60.672)
http://download.spotify.com/SpotifyFullSetup.exe (1.1.60.672)
are two different installer versions o.O

**Comment by OfficialEsco** [comment]
Actually, add the suggestion https://github.com/microsoft/winget-pkgs/pull/20701#discussion_r668021177, maybe it solves the issue

**Comment by ImJoakim** [comment]
> Hmm should we use the Offline installer instead? The /Silent switch somewhat works on it, wish we could stop the auto launch tho..

For sure, let's go with the offline installer. And yeah, it's a bit annoying that you're unable to stop the auto launch after install.

> Also just found out
> https://download.scdn.co/SpotifySetup.exe (1.1.63.568)
> https://download.scdn.co/SpotifyFullSetup.exe (1.1.63.568)
> and
> http://download.spotify.com/SpotifySetup.exe (1.1.60.672)
> http://download.spotify.com/SpotifyFullSetup.exe (1.1.60.672)
> are two different installer versions o.O

:o Interesting

**Comment by ImJoakim** [comment]
> Actually, add the suggestion [#20701 (comment)](https://github.com/microsoft/winget-pkgs/pull/20701#discussion_r668021177), maybe it solves the issue

Yeah, I think it will 🤞

**Comment by OfficialEsco** [comment]
I changed the Architecture to x86 because of this tho
image

@arjun-msft  @ChuckFerring https://github.com/microsoft/winget-pkgs/pull/14316#issuecomment-850886662
Whats the issue now? Don't tell me its not x86

**Comment by ImJoakim** [comment]
Seems like a lot of PR's are getting the `Internal-Error` label by looking at recent PRs.

**Comment by ImJoakim** [comment]
> Seems like a lot of PR's are getting the `Internal-Error` label by looking at recent PRs.

Yeah, seems  to be an issue with all new PRs @OfficialEsco

**Comment by OfficialEsco** [comment]
> Yeah, seems to be an issue with all new PRs @OfficialEsco

Yeeeah i was gonna say maybe try to reopen but then i checked a PR with a few comments and its a consistent issue and they are working on it 😁

**Comment by ImJoakim** [comment]
Seems like it needs SilentWithProgress as well.

```
PS C:\Users\WDAGUtilityAccount\Desktop\manifests> winget validate --manifest "C:\Users\WDAGUtilityAccount\Desktop\manifests\s\Spotify\Spotify\1.1.63.568.gda8cb5ac"
Manifest validation succeeded with warnings.
Manifest Warning: Silent and SilentWithProgress switches are not specified for InstallerType exe. Please make sure the installer can run unattended.
```

**Comment by OfficialEsco** [comment]
I don't actually know if the InstallerModes work so maybe, but there is a known issue with Spotify  https://github.com/microsoft/winget-pkgs/pull/14316#issuecomment-850886662 so i doubt...
Would be great if that was the issue for this case tho

**Review by OfficialEsco — APPROVED** [review]
@arjun-msft  @ChuckFerring https://github.com/microsoft/winget-pkgs/pull/14316#issuecomment-850886662

**Review by OfficialEsco — APPROVED** [review]

**Review by denelon — APPROVED** [review]

**Pull request merged**